bin/update-from-templates

```diff
@@ -38,6 +38,6 @@ for slug in $(ls exercises/practice); do
     fi
 done
 
-[[ ${num_updated} -eq 0 ]] || exit 2
+[[ ${num_updated} -eq 0 ]] || exit 1
```

Comment:
I changed the exit codes to be consistent with other pre-commit hooks/repos: 0 when nothing was changed, 1 if an expected change was made, 3 on any other error.